public/js/main.js

```diff
@@ -145,9 +145,78 @@ function displayText(data) {
 		container.appendChild(output);
 	}
 
+	setChildrenDataAttributes();
+
 	toggleLoadingState();
+  
+	document.getElementById("output").classList.remove("cape");
+  
+	formatChildElements();
+}
 
-	document.getElementById('output').classList.remove('cape');
+function formatChildElements() {
+	setElementsHeight();
+	window.addEventListener("resize", debounce(setElementsHeight, 250));
+}
+  
+function debounce(func, wait) {
+	var timeout;
+	return function() {
+	  var context = this,
```

Comment:
I have nothing against comma declaration, but it is standard at the FT that we explicitly declare variables, for readability.